### ISSUE TITLE
CSV is missing in the ext allowed filetypes 

### ISSUE THREAD

**Issue by bhatnagardivyanshu** [sindresorhus/file-type#167]
The filetypes allowed for the ext doesn't include csv which is commonly used

**Comment by MaxLeiter** [comment]
https://github.com/sindresorhus/file-type/issues/87 

This is only for supporting binary file types

**Comment by sindresorhus** [comment]
It's impossible to reliable detect CSV as it's just text. CSV is hard to parse, even if you know you're getting CSV, as there's no CSV standard and there are many variants, encodings, edge-cases, etc.
